salt/modules/alternatives.py

```diff
@@ -80,6 +80,25 @@ def show_current(name):
     return False
 
 
+def check_exists(name, path):
+    '''
+    Check if the given path is an alternative for a name.
+
```

Comment:
Sorry, thanks.  My comment was based on the original pull request and I didn't look over this one very thoroughly. :-)